public/src/main.tsx

```diff
@@ -34,7 +35,7 @@ const styles = ({ palette, spacing, mixins, typography }: Theme) => createStyles
   drawerPaper: {
     width: drawerWidth,
   },
-  toolbar: mixins.toolbar,
+  toolbar: mixins.toolbar as CSSProperties, // createStyles expects material-ui's CSSProperties type, not react's
```

Comment:
this is unrelated but has started causing the build to fail. I tried bumping the material-ui version but it didn't help. Casting makes the error go away